lib/matrix.gd

```diff
@@ -1221,16 +1250,20 @@ DeclareOperation( "TriangulizeMat", [ IsMatrix and IsMutable ] );
 ##
 ##  <Description>
 ##  returns a mutable list containing the entries of the <A>pos</A>th upper
-##  subdiagonal of <A>mat</A>.
+##  subdiagonal of the matrix <A>mat</A>.
```

Comment:
Offtopic (and not something I'd expect this PR to address!), but: what is a "subdiagonal"? Do we define this anywhere? I wouldn't be able to figure out what this function does from just that text, I'd have to look at examples (luckily there are some) and possibly the code.

And why don't we offer a "LowerSubdiagonal" command? (Or do we? If so, shouldn't its text be updated, too?)